src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -232,9 +232,8 @@ public int process(List<File> files) throws CheckstyleException {
         // It may also log!!!
         fileSetChecks.forEach(FileSetCheck::destroy);
 
-        final int errorCount = counter.getCount();
```

Comment:
Initially, I was wondering if inlining this could cause issues and whether it has been tested.